scripts/package-deploy.sh

```diff
@@ -1,15 +1,14 @@
 #!/bin/bash
-
 # stage-file copies the file $1 with the specified path $2
-# if no file exists it will silently continue
+# if no file exists it will abort
 function stage-file {
-    if [ -f $1 ]; then
-        echo "Copying $1 to $2"
-        cp $1 $2
-    else
+    if [ ! -f $1 ]
```

Comment:
would it be better to simply do `set -euxf -o pipefail` at the top, as we do in most our scripts?

Reply:
Yeah I'd agree with that.

Reply:
If you include `-x` then the echo for copying doesn't really make sense and the whole function can just be removed.